File: PRAK405/PRAK405-2210817310013-Ryan Muhammad Irfan.py
A,B = input().split()
A = int(A) ; B = int(B)
for i in range (1,A+1):
    jarak=i
    while jarak >1:
        print("(%d * %d) + "%(jarak,B), end='');jarak-=1
    o = 1 ; hasil=(i * B)
    while o<i:
        hasil += ( o * B) ; o+=1
    print("(%d * %d) = %d"%(jarak,B,hasil))
p=1 ; q=1 ;t = 0
while p<=A:
     t += q * B ; q += p + 1 ; p +=1
print("%d"%(t)) 

File: PRAK405/PRAK405-2210817310013-RyanMuhammadIrfan.c
#include <stdio.h>

int main()
{
    int X, Y, i, j, A, B, C, hasil, t;

    scanf("%d %d", &X, &Y);
    for (i = 1; i <= X; i++)
    {
        for (j = i; j > 1; j--)
        {
            printf("(%d * %d) + ", j, Y);
        }
        for (A = 1, hasil = A * Y; A < i; A++, hasil += (A * Y))
        {
            printf("(%d * %d) = %d\n", j, Y, hasil);
        }
        for (B = 1, C = 1, t = 0; B <= X; C += B + 1, B++)
        {
            t += C * Y;
        }
    }
    printf("%d", t);
}